### PR TITLE
Various fixes and improvements

### LINKED ISSUE #3: Doesn't work correctly in some MPRIS controllers (xfce4-pulaseaudio-plugin)
Hey,

I'm noticing that this doesn't quite work in some MPRIS controllers. For example xfce4-pulseaudio-plugin will always show "Not currently playing"

According to the Xfce guys (see [here](https://gitlab.xfce.org/panel-plugins/xfce4-pulseaudio-plugin/-/issues/98#note_80747) this is because no `PlaybackStatus` is sent. And indeed, using dbus inspectors that property seems to be missing.

Also it seems the "Next" controls are not working at all.

Thanks!

### PR DESCRIPTION
**Pull request by haarp**
Sup, I would like to merge some fixes and improvements. Making individual pull requests seems like a chore :upside_down_face: 

Fixes #3

### CODE REVIEW

**Comment by ryanswilson59** [comment]
I assume the way to test this is to run it in XFCE and verify that it works?

File: mediaplayer.py [hunk 1]
```diff
@@ -163,3 +163,5 @@ def SetPosition(self, TrackId, Position):
     @dbus.service.method("org.mpris.MediaPlayer2.Player", in_signature="s")
     def OpenUri(self, URI):
         pass
+
+# vim: set tabstop=4 expandtab
```
Comment (ryanswilson59) [inline mediaplayer.py]: I don't know much about using vim, is there a way that this can get moved to some config file?

File: pianobar-mediaplayer2 [hunk 2]
```diff
@@ -15,8 +16,9 @@ class pianoPlayer(mediaplayer.mediaPlayer):
         self.properties["org.mpris.MediaPlayer2.Player"]["CanPlay"] = True
         self.properties["org.mpris.MediaPlayer2.Player"]["CanPause"] = True
         self.properties["org.mpris.MediaPlayer2.Player"]["CanControl"] = True
+        self.properties["org.mpris.MediaPlayer2.Player"]["CanGoNext"] = True
         self.properties["org.mpris.MediaPlayer2"]["CanQuit"] = True
-        self.properties["org.mpris.MediaPlayer2"]["Identity"] = "PianoBar"
+        self.properties["org.mpris.MediaPlayer2"]["Identity"] = "pianobar"
```
Comment (ryanswilson59) [inline pianobar-mediaplayer2:21]: why was this change necessary?

**Comment by haarp** [comment]
Yup. I tested it and am using it myself on Xfce4 with xfce4-pulseaudio-plugin, but feel free to verify.

**Review by ryanswilson59 — COMMENTED** [review]
Overall looks good, I probably won't test the changes on xfce since I trust you got it to work for yourself.

File: mediaplayer.py (hunk 1 above)
Comment (haarp) [reply mediaplayer.py]: Modelines can be used by other editors too, I'm using xed.

I suppose I could also use an .editorconfig instead. Will make a commit.

File: pianobar-mediaplayer2 (hunk 2 above)
Comment (haarp) [reply pianobar-mediaplayer2:21]: No, it wasn't strictly necessary :upside_down_face: But the official spelling is lowercase, so I figured I'd use that.

Comment (ryanswilson59) [reply pianobar-mediaplayer2:21]: Sounds good then

File: mediaplayer.py (hunk 1 above)
Comment (haarp) [reply mediaplayer.py]: Pushed a new commit for an editorconfig.

**Comment by ryanswilson59** [comment]
LGTM!

**Comment by haarp** [comment]
Awesome, thanks!
